### PR TITLE
[tests-only][full-ci] Test for adding users to a group using invalid JSON

### PR DESCRIPTION
**Pull request by PrajwolAmatya**
## Description
This PR adds the tests for adding members in a group using invalid JSON.

## Related Issue

Related to issue: https://github.com/owncloud/ocis/issues/5569

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests only (no source changes)

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [x] Acceptance tests added
- [ ] Documentation ticket raised: <link>

### CODE REVIEW

File: tests/TestHelpers/GraphHelper.php [hunk 1]
```diff
@@ -608,6 +608,71 @@ public static function addUserToGroup(
...
+			$url,
+			$xRequestId,
+			'PATCH',
+			$adminUser,
+			$adminPassword,
+			self::getRequestHeaders(),
+			\json_encode($invalidJSON)
+		);
+	}
+
```
Comment (saw-jan) [inline tests/TestHelpers/GraphHelper.php:675]:
```suggestion
```

File: tests/acceptance/features/bootstrap/GraphContext.php [hunk 2]
```diff
@@ -2134,4 +2134,79 @@ public function theAdministratorAddGroupToAGroupAtOnceUsingTheGraphApi(string $u
...
+		$this->featureContext->setResponse(
+			GraphHelper::addUserToGroupWithInvalidJson(
+				$this->featureContext->getBaseUrl(),
+				$this->featureContext->getStepLineRef(),
+				$credentials["username"],
+				$credentials["password"],
+				$groupId,
+				$invalidJSON
+			)
+		);
```
Comment (saw-jan) [inline tests/acceptance/features/bootstrap/GraphContext.php:2172]:
```suggestion
		$response = HttpRequestHelper::sendRequest(
			self::getFullUrl($this->featureContext->getBaseUrl(), 'groups/' . $groupId);,
			$this->featureContext->getStepLineRef(),
			'PATCH',
			$credentials["username"],
			$credentials["password"],
			self::getRequestHeaders(),
			\json_encode($invalidJSON)
		);
		$this->featureContext->setResponse($response);
```

File: tests/acceptance/features/bootstrap/GraphContext.php [hunk 3]
```diff
@@ -2134,4 +2134,79 @@ public function theAdministratorAddGroupToAGroupAtOnceUsingTheGraphApi(string $u
...
+		$this->featureContext->setResponse(
+			GraphHelper::addUsersToGroupWithInvalidJson(
+				$this->featureContext->getBaseUrl(),
+				$this->featureContext->getStepLineRef(),
+				$credentials["username"],
+				$credentials["password"],
+				$groupId,
+				$invalidJSON
+			)
+		);
```
Comment (saw-jan) [inline tests/acceptance/features/bootstrap/GraphContext.php:2210]:
```suggestion$response = HttpRequestHelper::sendRequest(
			self::getFullUrl($this->featureContext->getBaseUrl(), 'groups/' . $groupId);,
			$this->featureContext->getStepLineRef(),
			'PATCH',
			$credentials["username"],
			$credentials["password"],
			self::getRequestHeaders(),
			\json_encode($invalidJSON)
		);
		$this->featureContext->setResponse($response);
```

**Review by saw-jan — COMMENTED** [review]
No need for other new methods because we will rarely use those

File: tests/acceptance/features/bootstrap/GraphContext.php [hunk 4]
```diff
@@ -2134,4 +2135,78 @@ public function theAdministratorAddGroupToAGroupAtOnceUsingTheGraphApi(string $u
...
+			GraphHelper::getFullUrl($this->featureContext->getBaseUrl(), 'groups/' . $groupId),
+			$this->featureContext->getStepLineRef(),
+			'PATCH',
+			$credentials["username"],
+			$credentials["password"],
+			['Content-Type' => 'application/json'],
+			\json_encode($invalidJSON)
+		);
+		$this->featureContext->setResponse($response);
+	}
```
Comment (amrita-shrestha) [inline tests/acceptance/features/bootstrap/GraphContext.php:2211]:
```suggestion
		$this->featureContext->setResponse(HttpRequestHelper::sendRequest(
			GraphHelper::getFullUrl($this->featureContext->getBaseUrl(), 'groups/' . $groupId),
			$this->featureContext->getStepLineRef(),
			'PATCH',
			$credentials["username"],
			$credentials["password"],
			['Content-Type' => 'application/json'],
			\json_encode($invalidJSON)
		));
```

File: tests/acceptance/features/bootstrap/GraphContext.php [hunk 5]
```diff
@@ -2134,4 +2135,78 @@ public function theAdministratorAddGroupToAGroupAtOnceUsingTheGraphApi(string $u
...
+
+		$response = HttpRequestHelper::post(
+			GraphHelper::getFullUrl($this->featureContext->getBaseUrl(), 'groups/' . $groupId . '/members/$ref'),
+			$this->featureContext->getStepLineRef(),
+			$credentials["username"],
+			$credentials["password"],
+			['Content-Type' => 'application/json'],
+			\json_encode($invalidJSON)
+		);
+		$this->featureContext->setResponse($response);
```
Comment (amrita-shrestha) [inline tests/acceptance/features/bootstrap/GraphContext.php:2172]:
```suggestion
		$this->featureContext->setResponse(HttpRequestHelper::post(
			GraphHelper::getFullUrl($this->featureContext->getBaseUrl(), 'groups/' . $groupId . '/members/$ref'),
			$this->featureContext->getStepLineRef(),
			$credentials["username"],
			$credentials["password"],
			['Content-Type' => 'application/json'],
			\json_encode($invalidJSON)
		));
```

**Review by SwikritiT — APPROVED** [review]
LGTM

**Review by saw-jan — APPROVED** [review]
LGTM

**Pull request merged**